libs/product/src/reducers/configurable-product-entities/configurable-product-entity.ts

```diff
@@ -1,5 +1,7 @@
+import { ID } from '@daffodil/core';
+
 export interface DaffConfigurableProductEntity {
-	id: string;
+	id: ID;
```

Comment:
I think this should be `DaffProduct['id']`

Reply:
changed